ci/prow/make_config.go

```diff
@@ -1315,9 +1315,13 @@ func addTestCoverageJobIfNeeded(jobDetailMap *map[string][]string, repoName stri
 }
 
 // generateSection generates the configs for the section with the given generator
-func generateSection(sectionName string, generator testgridEntityGenerator) {
+func generateSection(sectionName string, generator testgridEntityGenerator, skipReleasedProj bool) {
 	outputConfig(sectionName + ":")
 	for _, projName := range projNames {
+		// Do not handle the project if it is released and we want to skip it.
+		if skipReleasedProj && isReleased(projName) {
```

Comment:
why do we need skipReleasedProj? Is there a reason to not group them under a relase number if its released?

Reply:
We are handling these released repos in a following function `generateDashboardsForReleases` since they require completely different grouping logic - we want to group all repos with the same version under a single repo. 
Please refer to #669 for more details.